pkgs/top-level/python-packages.nix

```diff
@@ -285,6 +285,10 @@ in {
     pytestCheckHook = self.pytestCheckHook_6_1;
   };
 
+  aiohttp-apispec = callPackage ../development/python-modules/aiohttp-apispec {
+    webargs = self.webargs5;
```

Comment:
That is not allowed.

Reply:
Can you suggest an alternative?

Reply:
It should be done in the project that uses it and not in pythonPackages.

Reply:
I moved it into aiohttp-apispec, naming the arg webargs5. Fine?